src/sim/V2/swig/python/CMake/FindNumpy.cmake

```diff
@@ -1,44 +1,131 @@
-# Copyright (c) Lawrence Livermore National Security, LLC and other VisIt
-# Project developers.  See the top-level LICENSE file for dates and other
-# details.  No copyright assignment is required to contribute to VisIt.
+###############################################################################
```

Comment:
Looks like old, long copyright got added back in here

Reply:
- [ ] look at this